app/views/root/local_transaction.html.erb

```diff
@@ -58,3 +58,17 @@
     </div>
   </section>
 <% end %>
+
+<% if @postcode_error %>
+  <script type="text/javascript">
+    //<![CDATA[
```

Comment:
We don't need the `//<![CDATA` bits. See http://stackoverflow.com/questions/66837/when-is-a-cdata-section-necessary-within-a-script-tag for some details.

Is there a way we could avoid putting an inline script in the page?